doc/sphinx/proof-engine/tactics.rst

```diff
@@ -1676,11 +1639,40 @@ Controlling the proof flow
    .. exn:: @ident is used in conclusion.
       :undocumented:
 
-.. tacn:: generalize @term
-   :name: generalize
+   .. tacn:: specialize_eqs @ident
+      :undocumented:
+
+.. tacn:: hresolve_core ( @ident := @one_term ) {? at @nat_or_var } in @one_term
+   :undocumented:
+
+.. tacn:: hget_evar @nat_or_var
+   :undocumented:
+
+   .. todo: keep or remove destauto?
+
+.. tacn:: destauto {? in @ident }
+   :undocumented:
```

Comment:
Can you preserve the information on the commit that introduced this tactic in the comment? f3a53027589813ff19b3a7c46d84e5bd2fc65741

Reply:
I added the commit as a comment in the rst and included the description from the source with minor edits (a lousy description is better than none).